utils/python/CIME/system_test.py

```diff
@@ -365,6 +365,11 @@ def _create_newcase_phase(self, test):
                     mpilib = case_opt[1:]
                     create_newcase_cmd += " --mpilib %s" % mpilib
                     logger.debug (" MPILIB set to %s" % mpilib)
+                if case_opt.startswith('N'):
```

Comment:
We need to pass ninst to create_newcase because user_mods needs to know this value.